main.tf

```diff
@@ -1,6 +1,6 @@
 locals {
   enabled                 = module.this.enabled
-  enable_ecs_service_role = module.this.enabled && var.network_mode != "awsvpc" && length(var.ecs_load_balancers) <= 1
+  enable_ecs_service_role = module.this.enabled && var.network_mode != "awsvpc" && length(var.ecs_load_balancers) == 1
```

Comment:
@verbalius the original code enables the `enable_ecs_service_role` if the `length(var.ecs_load_balancers) <= 1` meaning if there aren't any load balancers.

Your change sets enables the `enable_ecs_service_role` if the number of load balancers is equivalent to 1 which conflicts with the PR title. Is this intended ? Could you explain ?

Reply:
Hi :)
I tried doing all the steps manually in the console and see
1. Task with bridge mode network
2.a. Service without LB (means length(var.ecs_load_balancers))
**Result**: You can't specify IAM role

![image](https://user-images.githubusercontent.com/26470717/124920387-f1201c00-dfff-11eb-9c8d-b763437b9bd0.png)
2.b Service with application LB for example
**Result**: Lets you choose a role

![image](https://user-images.githubusercontent.com/26470717/124920596-24fb4180-e000-11eb-8f3f-c4725ed9cec3.png)

So if we transfer it to expression:
if LB count = 0 this should be null:
`iam_role = local.enable_ecs_service_role ? coalesce(var.service_role_arn, join("", aws_iam_role.ecs_service.*.arn)) : null`
But how it is now `length(var.ecs_load_balancers)=0 <= 1 -> true`

Reply:
or if you want to try it yourself via terraform just try to create service with bridge network mode without load balancer.

Reply:
Why your commit contains: ```length(var.ecs_load_balancers) == 1``` ?

In our case we can define more than one load balancer for service..